.github/CODEOWNERS

```diff
@@ -1,13 +1,3 @@
 # The JS team owns everything by default
```

Comment:
I'd be good to update this comment to call out why smithy is also a codeowner.

Reply:
I just removed the comment entirely